source/includes/_queries.md

```diff
@@ -236,9 +233,9 @@ Transactions | an array of transactions for given account | Committed transactio
 
 ### Purpose
 
-When an information about transaction per user using certain asset is needed, `GetAccountAssetTransactions` query is sent.
+When an information about transaction per user who is using certain asset is needed, `GetAccountAssetTransactions` query is sent.
```

Comment:
As far as I understand, this query returns all transactions for a pair of asset and account. It is very hard to infer that from this sentence. Can we reword that, for example: "`GetAccountAssetTransactions` query returns all transactions associated with given account *and* asset"